fintrack/transactions/transaction_helper.py

```diff
@@ -0,0 +1,134 @@
+
+from . models import Transaction
+from accounts.models import Account
+from accounts.account_helper import AccountHelper
+
+
+WITHDRAWAL_CATEGORY = "withdrawal"
+
+
+class TransactionManager():
```

Comment:
I want you to find an other way for reconciliation of balance without changing previous transactions